packages/x-charts/src/internals/animation/useAnimate.test.tsx

```diff
@@ -280,47 +221,31 @@ describe('useAnimate', () => {
 
     const { rerender } = render(<TestComponent width={1000} skip={false} />);
 
-    await waitFor(
-      () => {
-        expect(calls).to.equal(2);
-      },
-      { interval: RAF_INTERVAL },
-    );
-    expect(lastCall).to.be.greaterThan(0);
-    expect(lastCall).to.be.lessThan(1000);
```

Comment:
Rather than waiting for a `RAF_INTERVAL` and expecting an exact value, which could change based on the processing power, we now expect the value to be at least the same or bigger than what we expect, though we don't really care how many times the render-function ran.

@bernardobelchior chould check if this change is a problem for testing the animation logic or not though.

Reply:
As long as the `lastCall` is correct, I think it's fine. Worst case scenario we won't catch if we start calling `applyProps` more frequently than needed, but it should be fine since it's expected to be a pure function that depends on the time delta. 

I wonder if we could wait for an animation frame instead. That way we wouldn't have this chance of introducing a regression, but I don't think it's a blocker.